content/en/agent/basic_agent_usage/aix.md

```diff
@@ -14,114 +14,86 @@ algolia:
 
 ## Installation
 
-A one-step ksh install script is provided on the [Agent download page][1] within Datadog. The script supports the following environment variables:
+### Prerequisites 
+- Root privileges (or sudo) on each host  
+- Outbound HTTPS (443) to `.datadoghq.com`  
+- curl / ksh (shipped on AIX by default)  
+- Verify your host meet the Supported AIX platforms 
 
-* **CHANNEL**: defaults to stable. Specifies the package repository channel.
-  * Values: `stable`, `beta`, `unstable`
-* **VERSION**: defaults to latest. Specifies the package version.
-* **PROXY**: defaults to none. Specifies the proxy URI.
-  * Example: `http://proxy.foo.com`
-* **PROXY_USER**: defaults to empty. Specifies the proxy server username.
-* **PROXY_PASSWORD**: defaults to empty. Specifies the proxy server password. For the process/container Agent, this variable is required for passing in an authentication password and cannot be renamed.
-* **INSECURE**: defaults to `false`. Allows skipping TLS validation.
+### Install Agent
 
-Alternatively, download links for the latest releases can be found on [this page][2].
+To install the Agent on AIX, follow the [in-app instructions in Fleet Automation][6], and run the generated script on your hosts.
 
-The installer may be executed as follows (as root):
+{{< img src="/agent/basic_agent_usage/aix_img_july_25.png" alt="The Datadog Agent installation step for hosts AIX." style="width:90%;">}}
 
-{{< code-block lang="shell" wrap="true" >}}
-installp -aXYgd ./datadog-unix-agent-<VERSION>.bff -e dd-aix-install.log datadog-unix-agent
-{{< /code-block >}}
 
-This installs the Agent in `/opt/datadog-agent`.
 
 ### Installation log files
 
 You can find the Agent installation log in the `dd-aix-install.log` file. To disable this logging, remove the `-e dd-aix-install.log` parameter in the installation command.
 
 ## Commands
 
-| Description                     | Command (as root)           |
-|---------------------------------|-----------------------------|
-| Start Agent as a service        | `startsrc -s datadog-agent` |
-| Stop Agent running as a service | `stopsrc -s datadog-agent`  |
-| Status of Agent service         | `lssrc -s datadog-agent`    |
-| Status page of running Agent    | `datadog-agent status`      |
-| Send flare                      | `datadog-agent flare`       |
-| Display command usage           | `datadog-agent --help`      |
-
-## Configuration
-
-The configuration files and folders for the Agent are located in `/etc/datadog-agent/datadog.yaml`
-
-A sample configuration file can be found in `/etc/datadog-agent/datadog.yaml.example`.
+| Description   | Command (as root)           |
+|---------------|-----------------------------|
+| Start Agent   | `startsrc -s datadog-agent` |
+| Stop Agent    | `stopsrc -s datadog-agent`  |
+| Status of Agent service | `lssrc -s datadog-agent`|
+| Status Agent service | `datadog-agent status`      |
```

Comment:
The overlap between `Status of Agent service` and `Status Agent service` might be confusing for some users. Would the previous description `Status page of running Agent` still apply? If so, it might be helpful to update this to clarify the difference.